lib/reverse_sentence.rb

```diff
@@ -1,6 +1,28 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(n)
 def reverse_sentence(my_sentence)
```

Comment:
This isn't working, but you have the process right.  I think you could get it working if you didn't split and make a copy of the original , you could instead reverse individual characters in the long string.